tests/snapshots/TestCLICommands_atmos_helmfile_apply_help.stdout.golden

```diff
@@ -17,23 +17,31 @@ Global Flags:
         --                          Use double dashes to separate Atmos-specific
                                     options from native arguments and flags for
                                     the command.
+                                    
+                                    
 
         --logs-file string          The file to write Atmos logs to. Logs can be
                                     written to any file or any standard file
                                     descriptor, including '/dev/stdout',
                                     '/dev/stderr' and '/dev/null' (default
                                     "/dev/stderr")
+                                    
+                                    
```

Comment:
Why are we adding 2 new line breaks?

Reply:
Now these line breaks only appear when we collect the output to a file.
Thanks to markdown renderer where we require two `\n` to get one.
So on cli this is visible as single `\n`. WIll post some pictures for clarification